FreestarAds-Ogury/Source/objc/OguryInterstitialMediator.m

```diff
@@ -27,17 +27,16 @@ - (BOOL)canShowInterstitialAd {
     return YES;
 }
 
-- (NSString*)placementId {
-    return self.mPartner.placement_id;
-}
-
 - (void)loadInterstitialAd {
     FSTRLog(@"OGURY: loadInterstitialAd");
     [self resetInterstitialAd];
 
-    self.ad = [[OguryInterstitialAd alloc] initWithAdUnitId:[self placementId]];
-    [self.ad load];
+    FSTRLog(@"OGURY: placement_id %@", self.mPartner.placement_id);
+    FSTRLog(@"OGURY: adunitId %@", [self.mPartner adunitId]);
+
+    self.ad = [[OguryInterstitialAd alloc] initWithAdUnitId:[self.mPartner adunitId]];
```

Comment:
Same here, should check adunit id for nil and if so, call _partnerAdLoadFailed:_

Reply:
Fixed